src/plugins/chinese/69xinshu.ts

```diff
@@ -150,7 +150,7 @@ class XinShu69 implements Plugin.PluginBase {
 
     const searchUrl = `${this.site}/modules/article/search.php`;
     const formData = new FormData();
-    // formData.append('searchkey', encode(searchTerm, 'gbk'));
+    formData.append('searchkey', encode(searchTerm, 'gbk'));
```

Comment:
Does the search work properly in app?

Reply:
I haven't had a chance to check it out yet

Reply:
Its domain was changed